examples/lang/erc20/src/lib.rs

```diff
@@ -168,10 +166,10 @@ mod tests {
 
     #[test]
     fn transfer_works() {
-        let alice = AccountId::try_from([0x0; 32]).unwrap();
-        let bob = AccountId::try_from([0x1; 32]).unwrap();
+        let alice: AccountId = From::from([0x0; 32]);
+        let bob: AccountId = From::from([0x1; 32]);
 
-        env::test::set_caller(alice);
+        env::set_caller(alice);
```

Comment:
So we kinda lost the `test` sub module here.
It arguably isn't that important but what are the chances we can reintroduce it because I kind of like the clear separation between environment and test-environmental functions.